source/assets/critical/c_landingpage.scss

```diff
@@ -9,7 +9,23 @@
 @import '../../_patterns/10-atoms/wrapper/ratio';
 @import '../../_patterns/40-templates/landingpage';
 
+@font-face {
+  font-family: 'SRG_SSR_Bold';
+  src: url('../fonts/SRGSSRTypeText_W_Bd.woff2') format('woff2');
```

Comment:
In den normalen Font-Deklarationen haben wir auch noch _woff_ drin – wird das hier nicht benötigt?

Reply:
ne ich denke nicht, alle modernen Browser sollten ja woff2 können. Ansonsten flackert es halt ein wenig :)